thrust/thrust/version.h

```diff
@@ -31,11 +31,13 @@
 #include <thrust/detail/config/config.h>
 
 #if defined(_CCCL_COMPILER_NVHPC) && defined(_CCCL_USE_IMPLICIT_SYSTEM_DEADER)
-#pragma GCC system_header
+#  pragma GCC system_header
 #else // ^^^ _CCCL_COMPILER_NVHPC ^^^ / vvv !_CCCL_COMPILER_NVHPC vvv
 _CCCL_IMPLICIT_SYSTEM_HEADER
 #endif // !_CCCL_COMPILER_NVHPC
 
+#include <cuda/version>
+
```

Comment:
Can we move this above the system header guard. I want to make sure that we blow up whenever we forget to use that guard. 

Only external (standard) headers should be after this